### PR TITLE
feat: trigger mail when user is mentioned in comment

### PR DESCRIPTION
**Pull request by gondar00**
fixes: #3442

### CODE REVIEW

**Review by gsmithun4 — CHANGES_REQUESTED** [review]

File: server/src/services/comment.service.ts [hunk 1]
```diff
@@ -5,16 +5,59 @@ import { Comment } from '../entities/comment.entity';
...
+          comment.comment,
+          mentionedUser.avatar?.data.toString('base64')
+        );
+        void this.commentUsersRepository.save(
+          this.commentUsersRepository.create({ commentId: comment.id, userId: mentionedUser.id })
+        );
+      }
+      return comment;
+    } catch (error) {
+      console.log(error);
```
[the remark this answers is not in the document]
Comment (gondar00) [reply server/src/services/comment.service.ts]: Updated with `InternalServerErrorException`

**Comment by shubh22** [comment]
Task linked: [CU-2h1aq0e Create comment notifications](https://app.clickup.com/t/2h1aq0e)

File: frontend/src/_ui/Mentions/index.js [hunk 2]
```diff
@@ -45,15 +52,20 @@ const Mentions = ({ users, value, setValue, placeholder, darkMode }) => {
...
+        setValue(newPlainTextValue);
+      }}
       placeholder={placeholder}
     >
       <Mention
         trigger="@"
-        regex={/@(\S+)/}
-        displayTransform={(display) => `@${display}`}
-        markup="(@__display__)"
+        // regex={/@(\S+)/}
```
Comment (gsmithun4) [inline frontend/src/_ui/Mentions/index.js:66]: Shall we remove the commented codes?

**Review by gsmithun4 — APPROVED** [review]
LGTM

**Pull request merged**